pcsx2/GameDatabase.cpp

```diff
@@ -873,7 +873,7 @@ void GameDatabaseSchema::GameEntry::applyGSHardwareFixes(Pcsx2Config::GSOptions&
 
 			case GSHWFixId::RecommendedBlendingLevel:
 			{
-				if (value >= 0 && value <= static_cast<int>(AccBlendLevel::Maximum) && static_cast<int>(EmuConfig.GS.AccurateBlendingUnit) < value)
+				if (GSIsHardwareRenderer() && value >= 0 && value <= static_cast<int>(AccBlendLevel::Maximum) && static_cast<int>(EmuConfig.GS.AccurateBlendingUnit) < value)
```

Comment:
This is not safe to access. `GSIsHardwareRenderer()` reads GS thread state.

If you want to do this, look at `EmuConfig.GS` instead.